.semaphore/semaphore.yml

```diff
@@ -9,7 +9,7 @@ global_job_config:
   prologue:
     commands:
       - checkout
-      - sem-version python 3.8
+      - sem-version python 3.9
```

Comment:
Why not use Python 3.11?

Reply:
It's set to 3.9 because that is the oldest version supported by the project. This will ensure that the project is compatible with 3.9 until we drop support for it.